ignite/pkg/cosmosaccount/cosmosaccount.go

```diff
@@ -51,7 +51,7 @@ const (
 
 // Registry for accounts.
 type Registry struct {
-	homePath           string
```

Comment:
I renamed this to avoid confusion. This is really the keyring dir, and was already used as such.